{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py

```diff
@@ -3,47 +3,38 @@
 import os
 
 from astropy.version import version as astropy_version
+
+# For Astropy 3.0 and later, we can use the standalone pytest plugin
 if astropy_version < '3.0':
```

Comment:
Do we really need the `if` here? Does `pytest_astropy_header` not work with `astropy<3`?

Reply:
This is following the setup recommended in https://github.com/astropy/pytest-astropy-header#migrating-from-the-astropy-header-plugin-to-pytest-astropy - basically the plugin indeed doesn't fully work with astropy < 3 so in that case we need to use the plugin that was bundled with astropy core (basically for astropy < 3 we can't disable the built-in one so we can't use the standalone plugin too)